dashboard/requirements.txt

```diff
@@ -3,4 +3,5 @@ psycopg2
 python-dotenv
```

Comment:
picky of me but maybe alphabetical so it's easier for someone to look at quickly and find what they want